package.json

```diff
@@ -23,6 +23,7 @@
     "@koa/cors": "^2.2.3",
     "camelize": "^1.0.0",
```

Comment:
@aklkv If we add `ember-cli-string-utils`, both `camelize` and `dasherize` should be removed.

Reply:
I think this is addressed in #14 I don't mind sticking with just camelize package tho